resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env ruby
```

Comment:
This is a totally custom script, it's basically converting the existing spec and control/rules/etc files into fpm commands.